### ISSUE TITLE
justevaled doesn't do anything

### ISSUE THREAD

**Issue by t0mpr1c3** [br3ttb/Arduino-PID-AutoTune-Library#8]
Seems redundant given that Runtime() returns a value anyway.

**Comment by rusefi** [comment]
Yep, this variable is never read - only written. Either it's really not needed and should be deleted, or maybe there is a bug in the code and it should be used somewhere? :)

**Comment by lupicia** [comment]
Maybe the Runtime()'s return value is used to judge if the PID parameters have been calculated. If the return value is 1, then we don't need to call Runtime() again. PS: I delete justevaled lol
